packages/components/src/RadioGroup/RadioGroup.css

```diff
@@ -22,7 +21,7 @@
   width: var(--space-base);
   height: var(--space-base);
   box-sizing: border-box;
-  margin: 0 var(--space-small) 0 0;
+  margin: var(--space-smallest) var(--space-small) 0 0;
```

Comment:
@manjuramesh09 I'm just curious why you are using `smallest` for the horizontal margin. Why not `small`?

Reply:
In order to match the text height. With small, the radio button position goes below the label text